content_script.js

```diff
@@ -17,6 +17,27 @@ class HTMLSerializer {
     this.FILTERED_TAGS = new Set(['script', 'noscript', 'style', 'link']);
 
     /**
+     * @private {Set<string>} Contains the tag names for elements
+     *     that have no closing tags.
```

Comment:
Great, thanks. Done.  In general, is one LGTM sufficient to merge, or should I wait for one from both you and Chris?